server/index.js

```diff
@@ -73,6 +74,52 @@ app.get('/api/get/waitlist', (req, res, next) => {
     })
     .catch(err => next(err));
 });
+
+app.get('/api/get/tables', (req, res, next) => {
+  const sql = `select * from "tables"
+              order by "tableNumber" asc`;
+  db.query(sql)
+    .then(result => {
+      const tableListResult = result.rows;
+      res.json(tableListResult);
+    });
+});
+
+app.put('/api/put/table/:id', (req, res, next) => {
+  const tableNumber = parseInt(req.params.id, 10);
+  const sql = `update "tables"
+      set     "userId" = $1
+      where   "tableNumber" = $2
+      returning *`;
+  const params = [null, tableNumber];
+  db.query(sql, params)
+    .then(result => {
+      const updatedRow = result.rows;
+      res.json(updatedRow);
+    })
+    .catch(err => next(err));
+});
+
+app.put('/api/put/table-assign/:id', (req, res, next) => {
+  const tableNumberfetch = parseInt(req.params.id, 10);
+  const { custId } = req.body;
+  const sql = `with "newCustomer" as (
+      update "tables" set "userId" = $1
+      where "tableNumber" = $2
+      returning "userId"
+      )
+      delete from "waitlist"
+      where "userId" = (select "userId" from "newCustomer")
+      returning *`;
+  const params = [custId, tableNumberfetch];
+  db.query(sql, params)
+    .then(result => {
+      const updatedRow = result.rows;
+      res.json(updatedRow);
```

Comment:
Since the client only updated one table, you should send the table object only and not an array.